src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -392,36 +433,22 @@ def create(dataset, target, feature, max_iterations=10,
             row_display_names.insert(2, 'Validation Accuracy (%)')
         table_printer = _tc.util._ProgressTablePrinter(row_ids, row_display_names)
 
-    train_metric = _get_accuracy_metric()
-    if validation_data:
-        validation_metric = _get_accuracy_metric()
-
     for i in range(max_iterations):
         # TODO: early stopping
-
-        for data, label in train_data:
-            custom_NN.train(data, label)
-        train_data.reset()
-
-        # Calculate training metric
         for data, label in train_data:
-            outputs = custom_NN.predict(data)
-            train_metric.update(label, outputs)
+            result = custom_NN.train(data, label)
+            train_accuracy = result['accuracy']
```

Comment:
How are we aggregating the accuracy across all the batches in the epoch?

Reply:
I replicated what we were doing previously in the MXNet case. i.e. `tf.reduce_means` returns the exact same output as `mx.metric.Accuracy()` on each iteration. i.e. the progress table is the same whether running the MXNet metric or TF's accuracy. 

If we want to aggregate accuracies I can add that feature.